helm_deploy/use-of-force/templates/ingress.yaml

```diff
@@ -10,7 +10,8 @@ metadata:
   annotations:
     kubernetes.io/ingress.class: "nginx"
     nginx.ingress.kubernetes.io/custom-http-errors: "418"
-    {{- toYaml .Values.ingress.annotations | nindent 4 }}
+    external-dns.alpha.kubernetes.io/set-identifier: use-of-force-{{ $fullName }}-green
```

Comment:
so using **use-of-force-{{ $fullName }}-green** seemed like the easiest way I could get the identifier to carry on being **use-of-force-use-of-force-${env}-green**

Reply:
meh.. guess you could have just hardcoded it